src/components/Homepage/LanguageMorpher.tsx

```diff
@@ -0,0 +1,72 @@
+"use client"
+
+import Morpher from "@/components/Morpher"
+import { Button } from "@/components/ui/buttons/Button"
+
+import {
+  DESKTOP_LANGUAGE_BUTTON_NAME,
+  HAMBURGER_BUTTON_ID,
+  MOBILE_LANGUAGE_BUTTON_NAME,
+} from "@/lib/constants"
+
+import { useMediaQuery } from "@/hooks/useMediaQuery"
+
+const LanguageMorpher = () => {
+  const handleMobileClick = () => {
```

Comment:
wow this is the first time I see that we have a button here xD (and a non-reactive approach to do it haha but understand that that was previous implementation)

Reply:
Yeah this was a tricky work-around since we're linking this button to a far-away elements that changes completely between the desktop nav bar and the mobile menu, so we needed a way to effectively programmatically trigger these menus. Subtle little feature/shortcut =)